front/src/applications/operationalStudies/components/BreadCrumbs.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+type Props = {
+  projectName?: string;
+  studyName?: string;
+  scenarioName?: string;
+};
+
+export default function BreadCrumbs({ projectName, studyName, scenarioName }: Props) {
```

Comment:
This needs to be a generic component, who can display N elements.

for example : 

```
const path = [
  { link: "/operational-studies", name: "t('projectsList')" }
  { link: "/operational-studies/project", name: projectName }
…
];

<Breadcrumb path={path} />
```

Reply:
The idea is neat, but there's not enough place on navbar, and it's by consequence, limited to project/study/scenario tree.
That's why it's not a common component but an operationalStudie's component.

Reply:
I prefer not to do this, just for the moment.